Sources/ProjectSpec/Scheme.swift

```diff
@@ -559,6 +593,9 @@ extension Scheme.Test.TestTarget: JSONEncodable {
         if parallelizable != Scheme.Test.TestTarget.parallelizableDefault {
             dict["parallelizable"] = parallelizable
         }
+        if let location = location {
+            dict["location"] = location
```

Comment:
This just puts the struct into the dictionary. Write the string
```suggestion
            dict["location"] = location.location
```